### PR TITLE
Review of general webpages documentation

### PR DESCRIPTION
**Pull request by ybouilla**
**PR description**
See #1042 

Corresponds to item:

> review and update the "general pages" web documentation (not tuto, not user guide) + developer web documentation + README files in the library (branch feature/896-docs-and-webpages-review )

**Developer Certificate Of Origin (DCO)**

By opening this merge request, you agree to the
[Developer Certificate of Origin (DCO)](https://github.com/fedbiomed/fedbiomed/blob/master/CONTRIBUTING.md#fed-biomed-developer-certificate-of-origin-dco)

This DCO essentially means that:

- you offer the changes under the same license agreement as the project, and
- you have the right to do that,
- you did not steal somebody else’s work.

**License**

Project code files should begin with these comment lines to help trace their origin:
```
# This file is originally part of Fed-BioMed
# SPDX-License-Identifier: Apache-2.0
```

Code files can be reused from another project with a compatible non-contaminating license.
They shall retain the original license and copyright mentions.
The `CREDIT.md` file and `credit/` directory shall be completed and updated accordingly.

**Guidelines for PR review**

General:

* give a glance to [DoD](https://fedbiomed.org/latest/developer/definition-of-done/)
* check [coding rules and coding style](https://fedbiomed.org/latest/developer/usage_and_tools/#coding-style)
* check docstrings (eg run `tests/docstrings/check_docstrings`)

Specific to some cases:

* update all conda envs consistently (`development` and `vpn`, Linux and MacOS)
* if modified researcher (eg new attributes in classes) check if breakpoint needs update (`breakpoint`/`load_breakpoint` in `Experiment()`, `save_state_breakpoint`/`load_state_breakpoint` in aggregators, strategies, secagg, etc.)
* if modified a component with versioning (config files, breakpoint, messaging protocol) then update the version following the rules in `common/utils/_versions.py`

### CODE REVIEW

File: docs/getting-started/getting-started.md [hunk 1]
```diff
@@ -196,7 +196,7 @@ training_args = {
     'dry_run': False,  
 }
 
-from fedbiomed.researcher.experiment import Experiment
+from fedbiomed.researcher.federated_workflows import Experiment
```
Comment (mvesin) [inline docs/getting-started/getting-started.md:199]: Nice pick @ybouilla 

But there are many other hanging `fedbiomed.researcher.experiment` (in docs, notebooks, docstrings)
Don't we want to remove them all in that branch ?

File: mkdocs.yml [hunk 2]
```diff
@@ -121,9 +121,8 @@ nav:
                 - Aggregators: './developer/api/researcher/aggregators.md'
                 - Datasets: './developer/api/researcher/datasets.md'
                 - Experiment: './developer/api/researcher/experiment.md'
-                - Jobs: './developer/api/researcher/jobs.md'
-                - Experiment: './developer/api/researcher/experiment.md'
```
Comment (mvesin) [inline mkdocs.yml:125]: I pushed a proposed misc fix for developer API documentation
- remove duplicate `Experiment` entry
- restore alphabetical order

File: docs/getting-started/getting-started.md (hunk 1 above)
Comment (mvesin) [reply docs/getting-started/getting-started.md:199]: To gain us time, I pushed a proposed commit to remove these occurrences. Is it ok for you ?

**Comment by ybouilla** [comment]
Hey Marc, thanks, but since this item of issue #1042 was focusing on non-tutorial material, i deliberately didnot update those.
I was updating those in another branch

**Comment by ybouilla** [comment]
Good for me, we can merge!

**Review by mvesin — COMMENTED** [review]
OK for me

**Pull request merged**